src/fragment-components/a-link.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { Link, Checkbox, TextInput } from 'carbon-components-react';
+import { css } from 'emotion';
+import { AComponent, ComponentInfo } from './a-component';
+import { ComponentCssClassSelector } from '../components/css-class-selector';
+
+import image from './../assets/component-icons/link.svg';
+import { angularClassNamesFromComponentObj, nameStringToVariableString, reactClassNamesFromComponentObj } from '../utils/fragment-tools';
+
+export const ALinkStyleUI = ({selectedComponent, setComponent}: any) => {
+	return <>
+		<TextInput
+			value={selectedComponent.text}
+			labelText='Text'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					text: event.currentTarget.value
+				});
+			}}
+		/>
+		<Checkbox
+			labelText='Disabled'
+			id='disable-label'
+			checked={selectedComponent.disabled}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					disabled: checked
+				});
+			}}
+		/>
+		<Checkbox
+			labelText='Inline'
+			id='Inline-select'
+			checked={selectedComponent.inline}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					inline: checked
+				});
+			}}
+		/>
+		<ComponentCssClassSelector componentObj={selectedComponent} setComponent={setComponent} />
+	</>
+};
+
+export const ALinkCodeUI = ({selectedComponent, setComponent}: any) => {
+	return <>
+		<TextInput
+			value={selectedComponent.codeContext?.href || ''}
+			labelText='href'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					codeContext: {
+						...selectedComponent.codeContext,
+						href: event.currentTarget.value
+					}
+				});
+			}}
+		/>
+	</>
+};
```

Comment:
```suggestion
	return <>
		<TextInput
			value={selectedComponent.codeContext?.name}
			labelText='Input name'
			onChange={(event: any) => {
				setComponent({
					...selectedComponent,
					codeContext: {
						...selectedComponent.codeContext,
						name: event.currentTarget.value
					}
				});
			}}
		/>
		<TextInput
			value={selectedComponent.codeContext?.href || ''}
			labelText='href'
			onChange={(event: any) => {
				setComponent({
					...selectedComponent,
					codeContext: {
						...selectedComponent.codeContext,
						href: event.currentTarget.value
					}
				});
			}}
		/>
	</>
};
```

We should also allow users to update the Input variable names (Ex. 'link2'Inline could become simpleLinkInline after adding the above input)